src/Hyperwallet/Model/TransferRefund.php

```diff
@@ -0,0 +1,285 @@
+<?php
```

Comment:
Please remove this file from your commit

Reply:
Done

Reply:
The file was remove earlier. Now the file is back after merging with the latest V3 branch.